### PR TITLE
Added Testing other elements 

### PR DESCRIPTION
**Pull request by Pavellys**

### CODE REVIEW

File: src/test/java/AddRemoveElementTest.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        driver.findElement(By.xpath("//*[@onclick='deleteElement()']")).click();
+
+        //Checking how many elements deleted
+        List<WebElement> elementsDelete = driver.findElements(By.xpath("//*[@onclick='deleteElement()']"));
+        int actualResultDelete = elementsDelete.size();
+
+        Assert.assertEquals(expectedResultDeleteElement, actualResultDelete);
+
+        driver.quit();
+
```
Comment (vchesnakoutest) [inline src/test/java/AddRemoveElementTest.java:46]: let's remove empty lines after last method call

File: src/test/java/CheckBoxesTest.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        List<WebElement> listBoxesChange = driver.findElements(By.xpath("//*[@type='checkbox']"));
+        boolean actualResult_1 = listBoxesChange.get(0).isSelected();
+        boolean actualResult_2 = listBoxesChange.get(1).isSelected();
+        //Compare results
+        Assert.assertEquals(expectedResult_1, actualResult_1);
+        Assert.assertEquals(expectedResult_2, actualResult_2);
+
+        driver.quit();
+
+
```
Comment (vchesnakoutest) [inline src/test/java/CheckBoxesTest.java:57]: let's remove empty lines after last method call

File: src/test/java/HoversTest.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        //Hover over the avatar and click on the link
+        action.moveToElement(avatar).moveToElement(driver.findElement(By.xpath("//*[@href='/users/1']")))
+                .click().build().perform();
+        //get actual result
+        String actualResult = driver.findElement(By.xpath("//h1")).getText();
+        //Compare expected result ans actual
+        Assert.assertEquals("not the correct result", expectedResult, actualResult);
+
+        driver.quit();
+
```
Comment (vchesnakoutest) [inline src/test/java/HoversTest.java:37]: let's remove empty lines after last method call

File: src/test/java/NotificationMessagesTest.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        actualResult = actualResult.substring(0,actualResult.length()-1);
+        //Compare expected and actual results
+        if(expectedResult_1.equals(actualResult)){
+            Assert.assertEquals(actualResult, expectedResult_1, "Message doesn't match");
+
+        }
+        else if(expectedResult_2.equals(actualResult)){
+            Assert.assertEquals(actualResult, expectedResult_1, "Message doesn't match");
+        }
+
```
Comment (vchesnakoutest) [inline src/test/java/NotificationMessagesTest.java:38]: please add driver.quit()

File: src/test/java/SortableDataTablesTest.java [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        Assert.assertEquals(expectedResult_2, actualResult_2);
+        String actualResult_3 = driver.findElement(By.xpath("//table[2]//tbody//tr[2]//td[1]")).getText();
+        Assert.assertEquals(expectedResult_3, actualResult_3);
+        String actualResult_4 = driver.findElement(By.xpath("//table[2]//tbody//tr[3]//td[2]")).getText();
+        Assert.assertEquals(expectedResult_4, actualResult_4);
+        String actualResult_5 = driver.findElement(By.xpath("//table[1]//tbody//tr[3]//td[5]")).getText();
+        Assert.assertEquals(expectedResult_5, actualResult_5);
+
+        driver.quit();
+
```
Comment (vchesnakoutest) [inline src/test/java/SortableDataTablesTest.java:42]: let's remove empty lines after last method call

File: src/test/java/TyposTest.java [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+        //Search for text elements
+        List<WebElement> textList = driver.findElements(By.tagName("p"));
+        //Receiving actual result
+        String actualResult = textList.get(1).getText();
+        //Compare expected result and actual
+        Assert.assertEquals(bug,expectedResult, actualResult);
+
+        driver.quit();
+
+
```
Comment (vchesnakoutest) [inline src/test/java/TyposTest.java:39]: let's remove empty lines after last method call

**Comment by Pavellys** [comment]
l

**Pull request merged**